shared/cloud-gateway/develop/_send-receive-media-streams-c.mdx

```diff
@@ -0,0 +1,540 @@
+import * as data from '@site/data/variables';
+
+<PlatformWrapper platform="linux">
+This page introduces how to use the Cloud Gateway to send media streams to the client and receive media streams from the client.
+
+## Prerequisites
+
+Before you begin, make sure you have downloaded the latest Cloud Gateway. See [Run the Sample Project](../get-started/integrate-c-sdk).
+
+## Preparation
+
+Complete the following steps before sending and receiving media streams.
+
+### Step 1: Initialize the SDK
+
+You need to call `createAgoraService` and `initialize` to create and initialize an `AgoraService` object. The `IAgoraService` object persists as long as the server app keeps running.
+
+The SDK supports user IDs in both int format and string format. This page only shows user IDs in int format (the character set can only be digits). To learn more about user IDs in string format, see [Using User IDs in String Format](../develop/stringuid-c).
+
+```C++
```

Comment:
Code highlighting looks off. Maybe something is wrong with the rendering.

Reply:
@FayHaifei Change C++ to `cpp` and see if this warning disappears when running `yarn start`
<img width="564" alt="image" src="https://user-images.githubusercontent.com/73678648/185885968-5f496419-9b71-4a6e-8294-f6d2c4a50ee3.png">

Reply:
Fixed.